dapr/clients/grpc/client.py

```diff
@@ -1120,6 +1123,66 @@ def wait(self, timeout_s: float):
                         raise e
                     time.sleep(min(1, remaining))
 
+    def get_metadata(self) -> GetMetadataResponse:
+        """Returns information about the sidecar allowing for runtime
+        discoverability.
+
+        The metadata API returns a list of the components loaded,
+        the activated actors (if present) and attributes with information
+        attached.
+
+        Each loaded component provides its name, type and version and also
+        information about supported features in the form of component
+        capabilities.
+        """
+        _resp, call = self._stub.GetMetadata.with_call(GrpcEmpty())
+        response: api_v1.GetMetadataResponse = _resp  # type alias
```

Comment:
This was done just for the sake of having type-hinting on the `response` object. An alternative way, as discussed on https://stackoverflow.com/questions/52082939/type-hints-when-unpacking-a-tuple, is to just declare `response`  type directly before its usage:

```python
response: api_v1.GetMetadataResponse 
response, call = self._stub.GetMetadata.with_call(GrpcEmpty())
```

Same number of lines but one less variable ;)

Reply:
I think this seems better and recommended way for unpacking tuples ...